### ISSUE TITLE
Migration and psql not compatible with 0.3.0

### ISSUE THREAD

**Issue by JerryHuang-LE** [pgjones/quart-db#9]
Hey code owner,

When I upgrade to latest version, I found out some compatible issues. I think it is better to fix that please.

1. Normal, the DATABASE_URL for Postgres db should be like `postgres://postgres:@127.0.0.1:5432/my_database`, so the schema here could be 
```
scheme == "postgresql" or "postgres"
```
https://github.com/pgjones/quart-db/blob/ffcb2c032747aeb1da8a21f39be1fbc90675fb06/src/quart_db/extension.py#L198

2. Can we add the connection here please? And sometimes it is convenient to do the database migration directly instead to start the services every time.
```
if self._backend is None:
    self._backend = self._create_backend()
```
https://github.com/pgjones/quart-db/blob/ffcb2c032747aeb1da8a21f39be1fbc90675fb06/src/quart_db/extension.py#L119

Thanks a lot!

**Comment by pgjones** [comment]
1. fixed in 550514b26187225c51eaaad7572c5e73e8446c4e.
2. I don't think this should be done within the migrate method as whatever creates the backend will need to disconnect as well, which can't be within the migrate method. What do you think about a cli command to run the migration?

**Comment by pgjones** [comment]
For part 2, I've added a `db-migrate` command in f3c847f552ca66cec168a3d5c3a0d48bdab49c61
